docs/docs/about/comparison-powershell.md

```diff
@@ -192,514 +192,533 @@ Update-SPODataEncryptionPolicy|Microsoft.Online.SharePoint.PowerShell|
+Resume-PnPWorkflowInstance|PnP.PowerShell|
+Revoke-PnPHubSiteRights|PnP.PowerShell|[spo hubsite rights revoke](../cmd/spo/hubsite/hubsite-rights-revoke.md)
+Revoke-PnPSiteDesignRights|PnP.PowerShell|[spo sitedesign rights revoke](../cmd/spo/sitedesign/sitedesign-rights-revoke.md)
+Revoke-PnPTenantServicePrincipalPermission|PnP.PowerShell|[spo serviceprincipal grant revoke](../cmd/spo/serviceprincipal/serviceprincipal-grant-revoke.md)
+Save-PnPClientSidePageConversionLog|PnP.PowerShell|
+Save-PnPProvisioningHierarchy|PnP.PowerShell|
+Save-PnPProvisioningTemplate|PnP.PowerShell|
+Save-PnPTenantTemplate|PnP.PowerShell|
+Send-PnPMail|PnP.PowerShell|[spo mail send](../cmd/spo/mail/mail-send.md)
+Set-PnPApplicationCustomizer|PnP.PowerShell|
+Set-PnPAppSideLoading|PnP.PowerShell|
+Set-PnPAuditing|PnP.PowerShell|
+Set-PnPAvailablePageLayouts|PnP.PowerShell|
+Set-PnPClientSidePage|PnP.PowerShell|[spo page set](../cmd/spo/page/page-set.md), [spo page header set](../cmd/spo/page/page-header-set.md)
+Set-PnPClientSideText|PnP.PowerShell|
+Set-PnPClientSideWebPart|PnP.PowerShell|
+Set-PnPContext|PnP.PowerShell|
+Set-PnPDefaultColumnValues|PnP.PowerShell|
+Set-PnPDefaultContentTypeToList|PnP.PowerShell|[spo list contenttype default set](../cmd/spo/list/list-contenttype-default-set.md)
+Set-PnPDefaultPageLayout|PnP.PowerShell|
+Set-PnPDocumentSetField|PnP.PowerShell|
+Set-PnPField|PnP.PowerShell|[spo field set](../cmd/spo/field/field-set.md)
+Set-PnPFileCheckedIn|PnP.PowerShell|[spo file checkin](../cmd/spo/file/file-checkin.md)
+Set-PnPFileCheckedOut|PnP.PowerShell|[spo file checkout](../cmd/spo/file/file-checkout.md)
+Set-PnPFolderPermission|PnP.PowerShell|
+Set-PnPFooter|PnP.PowerShell|
+Set-PnPGraphSubscription|PnP.PowerShell|
+Set-PnPGroup|PnP.PowerShell|
+Set-PnPGroupPermissions|PnP.PowerShell|
+Set-PnPHomeSite|PnP.PowerShell|[spo homesite set](../cmd/spo/homesite/homesite-set.md)
+Set-PnPHideDefaultThemes|PnP.PowerShell|[spo hidedefaultthemes set](../cmd/spo/hidedefaultthemes/hidedefaultthemes-set.md)
+Set-PnPHomePage|PnP.PowerShell|[spo web set](../cmd/spo/web/web-set.md)
+Set-PnPHubSite|PnP.PowerShell|[spo hubsite set](../cmd/spo/hubsite/hubsite-set.md)
+Set-PnPIndexedProperties|PnP.PowerShell|
+Set-PnPInPlaceRecordsManagement|PnP.PowerShell|[spo site inplacerecordsmanagement set](../cmd/spo/site/site-inplacerecordsmanagement-set.md)
+Set-PnPKnowledgeHubSite|PnP.PowerShell|[spo knowledgehub set](../cmd/spo/knowledgehub/knowledgehub-set.md)
+Set-PnPLabel|PnP.PowerShell|[spo list label set](../cmd/spo/list/list-label-set.md)
+Set-PnPList|PnP.PowerShell|[spo list set](../cmd/spo/list/list-set.md)
+Set-PnPListInformationRightsManagement|PnP.PowerShell|
+Set-PnPListItem|PnP.PowerShell|[spo listitem set](../cmd/spo/listitem/listitem-set.md)
+Set-PnPListItemAsRecord|PnP.PowerShell|[spo listitem record declare](../cmd/spo/listitem/listitem-record-declare.md)
+Set-PnPListItemPermission|PnP.PowerShell|
+Set-PnPListPermission|PnP.PowerShell|
```

Comment:
[list roleinheritance reset](https://pnp.github.io/cli-microsoft365/cmd/spo/list/list-roleinheritance-reset/)
[list roleinheritance break](https://pnp.github.io/cli-microsoft365/cmd/spo/list/list-roleinheritance-break/)

Reply:
lets also not do it here but in a separate issue #3349